examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -593,8 +622,8 @@ def transform_images(examples):
                 progress_bar.update(1)
                 global_step += 1
 
-                if accelerator.is_main_process:
-                    if global_step % args.checkpointing_steps == 0:
+                if global_step % args.checkpointing_steps == 0:
+                    if accelerator.is_main_process:
```

Comment:
I believe both of these could be clubbed together?

Reply:
True.